src/main/java/com/tictactoe/EasyComputerPlayer.java

```diff
@@ -4,11 +4,17 @@
     Board board;
     String symbol;
 
-    public EasyComputerPlayer(String symbol, Board board) {
-        this.symbol = symbol;
+    public EasyComputerPlayer() {
+    }
```

Comment:
If you are not taking any actions in the constructor, you can delete it. Java should construct your class properly by default.